src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -216,23 +216,41 @@ bool AlertManeuverRequest::PrepareResponseParameters(
 
 bool AlertManeuverRequest::IsWhiteSpaceExist() {
   LOG4CXX_AUTO_TRACE(logger_);
-  const char* str = NULL;
+  using smart_objects::SmartArray;
 
   if ((*message_)[strings::msg_params].keyExists(strings::tts_chunks)) {
-    const smart_objects::SmartArray* tc_array =
+    const SmartArray* tts_chuncks_arr =
```

Comment:
@LuxoftAKutsan 
Chunks

Reply:
@LuxoftAKutsan please fix typo

Reply:
@jacobkeeler @dev-gh Fixed in 7b57748b3213b6739f9e700c751f7a3b63c7f027